FWCore/Framework/src/EDConsumerBase.cc

```diff
@@ -76,6 +76,8 @@ EDConsumerBase::consumesCollector() {
 
 static const edm::InputTag kWasEmpty("@EmptyLabel@");
 
+static const std::string kCurrentProcess("@currentProcess");
```

Comment:
How about moving this to be a class static of `InputTag` just like `kSkipCurrentProcess`?